apps/ucp-ui/cypress/e2e/institutions.cy.ts

```diff
@@ -102,6 +104,52 @@ describe("institutions", () => {
       });
   });
 
+  it("renders institutions, sorts, checks order, sort again and checks order a second time", () => {
+    cy.loginWithoutWidgetRole();
+
+    const rowRegex = new RegExp(INSTITUTIONS_ROW_TEST_ID);
+
+    cy.waitForLoad();
+
+    cy.findAllByTestId(rowRegex)
+      .eq(0)
+      .invoke("attr", "data-testid")
+      .then((testIdBeforeSort) => {
+        cy.get(INSTITUTIONS_TABLE_SORT_PARENT_CLASS)
```

Comment:
Can't you just click on the header label?

Reply:
This is mainly my inexperience with cypress/testing-library. I'm open to simplified ways to do this. I got this idea from looking at other test files.

Reply:
I'll work on simplifying these selectors.